src/SkyCoords.jl

```diff
@@ -66,6 +67,22 @@ const GAL_TO_FK5J2000 = FK5J2000_TO_GAL'
 const GAL_TO_ICRS = FK5J2000_TO_ICRS * GAL_TO_FK5J2000
 const ICRS_TO_GAL = GAL_TO_ICRS'
 
+
+# Gal --> SuperGal
+# we use the same parameters as in astropy
+sgp_l = deg2rad(47.37)
+sgp_b = deg2rad(6.32)
+# rotation matrix compared to astropy
+const GAL_TO_SUPERGAL = RotZYZ(-π / 2, sgp_b - π / 2, -sgp_l)'
+const SUPERGAL_TO_GAL = GAL_TO_SUPERGAL'
+# SuperGal --> ICRS: chain through GAL
+const SuperGal_TO_ICRS = GAL_TO_ICRS * SUPERGAL_TO_GAL
+const ICRS_TO_SuperGAL = SuperGal_TO_ICRS'
+# SuperGal --> FK5J2000: chain through GAL
+const SuperGal_TO_FK5J2000 = GAL_TO_FK5J2000 * SUPERGAL_TO_GAL
+const FK5J2000_TO_SuperGAL = SuperGal_TO_FK5J2000'
+
+
```

Comment:
Hello! After reviewing your code, I think I've been able to come up with fixes
After reviewing the output and source code for `astropy.coordinates.builtin_frames.supergalactic_transforms.supergal_to_gal()`, I found the proper rotation matrix for `GAL_TO_SUPERGAL`. Please change lines 71–84 in src/SkyCoords.jl to the following snippet.
```suggestion
# Gal --> SuperGal
# we use the same parameters as in astropy
sgp_l = deg2rad(47.37)
sgp_b = deg2rad(6.32)
# rotation matrix compared to astropy
const GAL_TO_SUPERGAL = RotZYZ(π/2, π/2 - sgp_b, π - sgp_l)
const SUPERGAL_TO_GAL = GAL_TO_SUPERGAL'
# SuperGal --> ICRS: chain through GAL
const SUPERGAL_TO_ICRS = GAL_TO_ICRS * SUPERGAL_TO_GAL
const ICRS_TO_SUPERGAL = SUPERGAL_TO_ICRS'
# SuperGal --> FK5J2000: chain through GAL
const SUPERGAL_TO_FK5J2000 = GAL_TO_FK5J2000 * SUPERGAL_TO_GAL
const FK5J2000_TO_SUPERGAL = SUPERGAL_TO_FK5J2000'
```